platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetMetrics.java

```diff
@@ -40,15 +46,37 @@ public class TipsetMetrics {
                     + "node's events as other parents.");
     private final RunningAverageMetric bullyScoreMetric;
 
+    private final Map<NodeId, SpeedometerMetric> parentMetrics = new HashMap<>();
+    private final Map<NodeId, SpeedometerMetric> pityParentMetrics = new HashMap<>();
+
     /**
      * Create metrics for the tipset event creator.
      *
      * @param platformContext the platform context
      */
-    public TipsetMetrics(@NonNull final PlatformContext platformContext) {
+    public TipsetMetrics(@NonNull final PlatformContext platformContext, @NonNull final AddressBook addressBook) {
+
         final Metrics metrics = platformContext.getMetrics();
         tipsetScoreMetric = metrics.getOrCreate(TIPSET_SCORE_CONFIG);
         bullyScoreMetric = metrics.getOrCreate(BULLY_SCORE_CONFIG);
+
+        for (final Address address : addressBook) {
+            final NodeId nodeId = address.getNodeId();
+
+            final SpeedometerMetric.Config parentConfig = new SpeedometerMetric.Config(
+                            "platform", "otherParent" + nodeId.id())
+                    .withDescription("Cycled when this node has used an event from this node as a "
```

Comment:
These descriptions need a rewording - you're using the phrase `this node` to refer to both "this" node *and* "that" node

Reply:
changed to `when this node has used an event from the target node node`